packages/beacon-node/test/spec/presets/fork_choice.test.ts

```diff
@@ -180,7 +180,7 @@ const forkChoiceTest =
                 });
                 if (!isValid) throw Error("Expect error since this is a negative test");
               } catch (e) {
-                if (isValid) throw e;
+                if (isValid || (e as Error).message === "Expect error since this is a negative test") throw e;
```

Comment:
it's better to have this string as a constant or make a specific error code for this but I think this is good to go